maintenance_plan_activity/tests/test_maintenance_plan_activity.py

```diff
@@ -49,11 +49,11 @@ def test_01_cron_auto_create_activities(self):
         # Check if activity Call has been created for the request 1
         generated_activities = self.mail_activity_obj.search(
             [('res_id', '=', request_1.id)])
-        self.assertEqual(len(generated_activities), 1)
+        self.assertEqual(len(generated_activities), 2)
```

Comment:
Why there is two activities?

Reply:
In v12, an activity " Maintenance Request" is created, that is why there are 2 activities.
![Selection_090](https://user-images.githubusercontent.com/36236518/71908842-08621d00-316f-11ea-8995-03a86d6e2143.png)

Reply:
Thanks!! I didn't know :wink: